src/components/DataTable/DataTable.less

```diff
@@ -0,0 +1,31 @@
+.lucid-DataTable {
+	&-message-container {
+		max-width: 401px;
+		background-color: @color-white;
+		border: @border-lightBorder;
+		border-radius: @size-borderRadius;
+	}
+
+	&-message-contents {
+		padding: @size-S @size-standard @size-standard @size-standard;
+	}
+
+	&-message-title {
+		color: @color-darkGray;
+		display: block;
+		font-size: 15px;
```

Comment:
I did a quick code scan and it doesn't appear any of our internal consumers are using `@size-font-large` so we should be okay to rename it without breaking anything.